csi/controller_server.go

```diff
@@ -164,8 +164,11 @@ func (cs *ControllerServer) ControllerPublishVolume(ctx context.Context, req *cs
 
 	logrus.Debugf("ControllerPublishVolume: current nodeID %s", req.GetNodeId())
 	if needToAttach {
-		// attach longhorn volume
-		input := &longhornclient.AttachInput{HostId: req.GetNodeId()}
+		// attach longhorn volume with frontend enabled
+		input := &longhornclient.AttachInput{
+			HostId:          req.GetNodeId(),
+			DisableFrontend: false,
```

Comment:
Note: if we don't pass any parameters, the frontend should be enabled by default.